transports/onion/src/lib.rs

```diff
@@ -0,0 +1,232 @@
+#![doc(html_logo_url = "https://libp2p.io/img/logo_small.png")]
+#![doc(html_favicon_url = "https://libp2p.io/img/favicon.png")]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+//! Tor based transport for libp2p. Connect through the Tor network to TCP listeners.
+//!
+//! Main entrypoint of the crate: [OnionTransport]
```

Comment:
```suggestion
//! Main entrypoint of the crate: [`OnionTransport`]
```

This will make sure rustdoc flags it as a bad link if it is ever renamed.